happy-birthday-to-you.js

```diff
@@ -0,0 +1,153 @@
+/* 
+
+------ STEPS ------
+1.  Ensure that name is in correct format (only strings, also account for 
+	special characters).
+
+2. Once correct name format is inputted, compare it to object key/value pairs.
+
+2a. Play a birthday song if dates match. otherwise, play the Rick Roll 
+
+*/
+
+const birthdayObj = {
+
+	'Abdullah Sayeem': 513,
+	'Andre Chow': 103,
+	'Gurdeep Singh': 327,
+	'Jimmy Feng': 711,
+	'John Chen': 1202,
+	'Kevin Chen': 627,
+	'Kevin Cheung':	521,
+	'Mohammed Rahman': 305,
+	'Robert Tan': 523,
+	'Roland Zhou': 903,
+	'Vainius Glinskis': 805,
+	'Zarif Rahat': 1107,
+
+};
+
+function validateInputName() {
+	let text = nameInput.value.toLowerCase();
+	// any combination of letters separated by a whitespace
+	let nameFormat = /^[a-zA-Z]+\s+[a-zA-Z]+$/; 
+
+	return nameFormat.test(text);
+}
+
+function confirmOnList() {
+	let text = nameInput.value.toLowerCase();
+
+	for (let name in birthdayObj) {
+		name = name.toLowerCase();
+
+		if (text === name) {
+			return true;
+		}
+	}
+
+	return false;
+}
+
+function getCurrentDate() {
+	const today = new Date();
+	const dd = today.getDate();
+	const mm = today.getMonth() + 1; // January is 0!
+	const yyyy = today.getFullYear();
+
+	return `${mm}${dd}`; 
+}
+
+const nameInputContainer = document.querySelector('.name-input-container');
+const nameInput = document.getElementById('name-input');
+const mainContent = document.querySelector('.main-content');
+const goButton = document.getElementById('go-button');
+
+goButton.addEventListener('click', function(event) {
+
+	// remove last child of both `<div>`s every time go button is clicked*
+	if (nameInputContainer.lastChild.nodeName === 'H6') {
+		nameInputContainer.lastChild.remove();
+	}
+
+	if (nameInputContainer.lastChild.nodeName === 'H1') {
+		mainContent.lastChild.remove();
+	}
```

Comment:
it's actually quite expensive to remove DOM nodes like this.

A faster alternative would be to simply always have these nodes there, but just let them start as empty. And then you just remove the content or add the content as needed.